README.md

```diff
@@ -1,6 +1,8 @@
 # PyoMyo
 Python module for the Thalmic Labs Myo armband. 
 
+## Test on Wednesday 10/20/2021
+
```

Comment:
test of reactions and tags fix not secure

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Not secure

Reply:
The software constructs all or part of an SQL command using externally-influenced input from an upstream component, but it does not neutralize or incorrectly neutralizes special elements that could modify the intended SQL command when it is sent to a downstream component.

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix